test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -331,12 +335,21 @@ void createReaders(EventStreamClientFactory clientFactory, String readerGroupNam
     }
 
     void createWriters(EventStreamClientFactory clientFactory, final int writers, String scope, String stream) {
```

Comment:
Can we use `TestState.txnWrite ` boolean to decide between txn v/s normal writers when `createWriters` are invoked?
All these tests do a `testState = new TestState(true);` in the @Setup method , which sets the `txnWrite` to true.

Reply:
Good point, @shrids. Yes, I will better use that flag which already is there to decide the type of writers to instantiate.